### PR TITLE
Update binaries and readme

### PR DESCRIPTION
**Pull request by sadikovi**
This PR updates `parquet-schema` and `parquet-read` and adds `Binaries` section to the main README, so people can install the scripts and use them for Parquet files (tested locally with `cargo install`).

### CODE REVIEW

**Comment by sadikovi** [comment]
@sunchao Could you review this PR when you have time? Thanks!

File: README.md [hunk 1]
```diff
@@ -27,6 +27,16 @@ Run `cargo build` or `cargo build --release` to build in release mode.
...
 
+## Binaries
+The following binaries are provided (use `cargo install` to install them):
+- **parquet-schema** for printing Parquet file schema and metadata.
+`Usage: parquet-schema <file-path> [verbose]`, where `file-path` is a path to the Parquet file,
+and optional `verbose` allows to print full metadata.
+
+- **parquet-read** for reading records from a Parquet file.
+`Usage: parquet-read <file-path> [num-records]`, where `file-path` is a path to the Parquet file,
+and optional `num-records` is a number of records to read from a file.
```
Comment (sunchao) [inline README.md]: nit:  `is a number` -> `is the number`. Also perhaps point out that when `num-records` is not specified, all records will be printed.

Comment (sadikovi) [reply README.md]: Yes, will do. Thanks.

**Comment by sadikovi** [comment]
@sunchao I updated the README file. Could you have a look again? I corrected the grammar, and added a note for on default behaviour. Thanks!

**Comment by sunchao** [comment]
Thanks @sadikovi ! Merged.

**Comment by sadikovi** [comment]
@sunchao Thanks!

**Pull request merged**